genkit-tools/common/src/manager/manager.ts

```diff
@@ -110,6 +117,14 @@ export class RuntimeManager {
         );
   }
 
+  subscribeToRuntimeEvents(
```

Comment:
It would be `manager.subscribeToRuntimeEvents(...)` which makes it seem like the manager is subscribing. Maybe just `onRuntimeEvents`? Also please add a JSDoc explaining how to use it.

Reply:
done!